addons/point_of_sale/report/pos_invoice.py

```diff
@@ -23,4 +23,7 @@ def get_report_values(self, docids, data=None):
             not_invoiced_orders_names = [a.name for a in not_invoiced_posorders]
             raise UserError(_('No link to an invoice for %s.') % ', '.join(not_invoiced_orders_names))
 
-        return self.env['ir.actions.report'].sudo().render_template('account.report_invoice', {'docs': self.env['account.invoice'].sudo().browse(ids_to_print)})
+        return {
+            'docs': self.env['account.invoice'].sudo().browse(ids_to_print),
+            'template_name': 'account.report_invoice'
+        }
```

Comment:
change name of report to "_name = 'report.account.report_invoice'" and on the action: 
<field name="report_name">account.report_invoice</field>